### DISCUSSION TITLE
"Server Error (500)" trying to activate organization plan

### DISCUSSION THREAD

**Discussion by woutegging** [opengisch/QField#3942] [Q&A]
Hello, 

I'm trying to activate an organization plan but I'm getting a "Server Error (500)". 
I've already send an email to hello@qfield.cloud but I didn't get an answer so far (email send on 14-02-2023). I already tried it on different webbrowsers. 
Does anyone know how to solve this problem?

Thank you!

Best regards, 

Wout

**Comment by suricactus** [comment]
@woutegging Thanks for this post. We will check why the email conversation didn't work out and continue by email.

Will also investigate the issue you are reporting. Note that if you cannot upgrade due to technical reasons, you are still eligible for the discounts ;) .

**Reply by woutegging** [reply]
@suricactus thank you!

**Reply by AresDraco** [reply]
Same problem, just tried to activate the plan and got the same error.

**Reply by suricactus** [reply]
@woutegging and @AresDraco can you please share your usernames on QFieldCloud?

**Reply by suricactus** [reply]
@AresDraco I assume your username is `aresdraco`. I think you can attempt an upgrade now.

@woutegging and @AresDraco the probem has been identified and will be fixed by tomorrow. Both get in touch with us on beta-transition@qfield.cloud with the URL of this discussion in it.

**Reply by woutegging** [reply]
My username is woutegging in QFieldCloud

I'll send you an email, thank you so far!
